src/citrine/_rest/resource.py

```diff
@@ -46,6 +48,49 @@ def access_control_dict(self) -> dict:
         }
 
 
+GEMDSelf = TypeVar('GEMDSelf', bound='GEMDResource')
+
+
+class GEMDResource(Resource[GEMDSelf]):
```

Comment:
All GEMD objects / DictSerializables go through this one Resource to maintain a consistent interface layer.